assets/src/components/detours/diversionPage.tsx

```diff
@@ -123,7 +123,13 @@ export const DiversionPage = ({
               Detour is not editable from this screen.
             </Alert>
           )}
-          {routingError && <RoutingErrorAlert />}
+          {routingError?.type === "no_route" && (
+            <RoutingErrorAlert>
+              You can&apos;t route to this location. Please try a different
+              point.
```

Comment:
nit(non-blocking): I almost want to wrap this like
```suggestion
              You can&apos;t route to this location.
              Please try a different point.
```

Just to get github to display it nicely 😭 
![image](https://github.com/mbta/skate/assets/6913198/a46a9e02-7338-4ed8-b12f-adfbcd8f2c7f)

Reply:
If it has to wrap in the browser, it doesn't look good.

<img width="1031" alt="Screenshot 2024-04-17 at 2 20 44 PM" src="https://github.com/mbta/skate/assets/912020/359ed277-6459-47f3-af0a-347d690364bb">

Probably not a huge deal, especially since I imagine this will mostly be used on screens that are large enough to avoid text-wrapping, but still.

Reply:
I'm not sure eager wrapping is much better... 
![image](https://github.com/mbta/skate/assets/6913198/13d8684a-a540-4154-ad24-8f2fd94eb0fd)